modules/tile-converter/src/i3s-converter/helpers/node-debug.ts

```diff
@@ -66,6 +103,7 @@ function getTileObbVertices(node) {
       (positions[i + 2] *= halfSize[2])
     );
     const rotatedPositions = positionsVector
+      // @ts-ignore
```

Comment:
It is recommended to use ts-expect-error instead of ts-ignore. When the upstream types are fixed, these comments will be identified.

Reply:
Done